infra/modules/worklytics-connector-specs/google-workspace.tf

```diff
@@ -0,0 +1,150 @@
+locals {
+
+  google_workspace_example_user = coalesce(var.google_workspace_example_user, "REPLACE_WITH_EXAMPLE_USER@YOUR_COMPANY.COM")
+  google_workspace_example_admin = coalesce(var.google_workspace_example_admin, var.google_workspace_example_user, "REPLACE_WITH_EXAMPLE_ADMIN@YOUR_COMPANY.COM")
+  google_workspace_sources = {
+    # GDirectory connections are a PRE-REQ for gmail, gdrive, and gcal connections. remove only
+    # if you plan to directly connect Directory to worklytics (without proxy). such a scenario is
+    # used for customers who care primarily about pseudonymizing PII of external subjects with whom
```

Comment:
for this use case we'd need rules to only pseudonymize emails NOT IN domain list.